pyigm/guis/igmguesses.py

```diff
@@ -1421,13 +1435,13 @@ def __init__(self, component=None, parent=None):
         self.parent = parent
         #if not status is None:
         #    self.statusBar = status
-        self.label = QtGui.QLabel('Component:',self)
+        self.label = QLabel('Component:',self)
         self.zwidget = ltgsm.EditBox(-1., 'zc=', '{:0.5f}')
```

Comment:
Also, if I somewhat remove all the values, i.e. leaving only the string ``""``, the GUI breaks because cannot convert that string to a float. The same happens if I accidentally add a string that would make ``float(str)`` fail.
Update: I added a commit that deals with this, but the first issue remains (see previous comment).